utils/scripts/update_generated_code_from_idl.sh

```diff
@@ -2,6 +2,8 @@
 
 files_to_exclude=(
     './include/fastrtps/types/*'
+    './include/fastdds/dds/xtypes/type_representation/*'
+    './include/fastdds/dds/core/detail/DDSReturnCode.idl'
```

Comment:
After talking with Ricardo, we have agreed that it is better to generate the code twice if necessary in case that the dependency is removed later on, and then the generated code would never be updated.